src/zkp/pimod.rs

```diff
@@ -781,7 +781,7 @@ mod tests {
         let (decryption_key, p, q) = DecryptionKey::new(rng).unwrap();
 
         let input = PiModInput {
-            N: decryption_key.encryption_key().n().to_owned(),
+            N: decryption_key.encryption_key().modulus().to_owned(),
```

Comment:
nit: the `encryption_key()` call could go away now, since we can access the modulus directly from decryption key. Feel free to postpone this to #50, though.

Reply:
Good call. I'll leave it for #50 though.